.github/updatecli/manifest-major.yaml

```diff
@@ -3,106 +3,48 @@ sources:
   common:
     kind: helmchart
     spec:
-      url: "https://charts.bitnami.com/bitnami"
-      name: "common"
-      version: "*.*.*"
+      url: https://charts.bitnami.com/bitnami
+      name: common
+      versionFilter:
+        kind: semver
+        pattern: 2.x.x
     sourceid: common
   postgresql:
     kind: helmchart
     spec:
-      url: "https://charts.bitnami.com/bitnami"
-      name: "postgresql"
-      version: "*.*.*"
-    sourceid: postgresql
-  helm:
-    kind: githubRelease
-    spec:
-      owner: helm
-      repository: helm
-      token: '{{ requiredEnv "GITHUB_TOKEN" }}'
-      versionFilter:
-        kind: semver
-        pattern: "x.x.x"
-    transformers:
-      - trimprefix: "v"
-  kind:
```

Comment:
these were managing the version of helm/kind to specify in the lint/test workflows. Since we were always updating to latest, removing this and just allowing the default version to apply.